executor/load_data.go

```diff
@@ -606,7 +626,10 @@ func (e *LoadDataInfo) ReadRows(ctx context.Context, parser mydump.Parser) error
 			if errors.Cause(err) == io.EOF {
 				return nil
 			}
-			return err
+			return ErrLoadDataCantRead.GenWithStackByArgs(
+				err.Error(),
+				"Only the following formats delimited text file (csv, tsv), parquet, sql are supported. Please provide the valid source file(s)",
```

Comment:
should we include help message in [doc](https://docs.pingcap.com/tidb/dev/error-codes), not in error message? seems most tidb error don't have msg for how to handle it.

Reply:
I'll discuss with PM about it later